### ISSUE TITLE
monaco-editor/min/vs/editor/editor.main.css require external fonts

### ISSUE THREAD

**Issue by Betalos** [microsoft/monaco-editor#1759]
**monaco-editor version:** 0.19.2
the editor.main.css now in 0.19.* requires an external file to be present not like 0.18.* wich bundle it with the file as an url.

**Comment by alexdima** [comment]
Indeed, since 0.19.x the Monaco Editor uses a font to render icons.

Currently, the font is not embedded as an inline URL in the CSS, mainly because base64 encoding it would increase the font file size by about 1.5 times.

**Comment by Betalos** [comment]
@alexdima in that case I would sudgest to include a scss file with the ability to use vars to point to the file (ease of integration with builders webpack, rollup etc)

**Comment by alexdima** [comment]
@Betalos AFAIK bundlers such as webpack look inside the CSS to load all required resources and fail/throw if something is missing. That is why I had to do e.g. https://github.com/microsoft/monaco-editor-webpack-plugin/commit/eaacb00bb4d97d7d26d303c0b761d5016be6e81f to instruct webpack what to do with the `.ttf` it finds automatically. 

In other words, I didn't need to tell webpack about the ttf, because webpack looked by itself inside the CSS.

**Comment by hediet** [comment]
It seems like this is not a feature request we are going to address soon, thus closing it.

> How do I Monaco default font to a font of my own choice?

Please ask this on stack overflow!
